### ISSUE TITLE
Promotion on Facebook

### ISSUE THREAD

**Issue by piranna** [NodeOS/NodeOS#282]
Create a Facebook account that would allow to link Twitter updates on the Facebook NodeOS page.

**Comment by tylerl0706** [comment]
Does this just mean fb would mirror what's posted on Twitter?

**Comment by piranna** [comment]
Yes. Maybe we could publish something there, but mostly just only what's on
Twitter. Just a way to get diffusion using Likes.

**Comment by tylerl0706** [comment]
How about:

https://ifttt.com/recipes/114287-when-i-tweet-it-then-feeds-to-my-facebook-fan-page

**Comment by piranna** [comment]
I tried to use native Twitter integration but give me error because my
Facebook wall is already integrated. I checked iftt and didn't asked me to
provide the page, so I think it would do the same... I'll check later on
Facebook what happened.
